debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
I believe this should be: `3.0 (native)` since the debian/ directory is part of the same source tree as the source code itself; the source isn't part of a separate orig.tar file.

Reply:
(Fixing that allows me to remove `-b` from the `dpkg-buildpackage` command you gave me)